tests_e2e/playwright/test_playwright.py

```diff
@@ -0,0 +1,16 @@
+from run_notebook import RunNotebook
+
+
+def test_notebook(navigator):
+    test_app = RunNotebook(navigator=navigator)
+    test_app.nav.clone_repo(
```

Comment:
Why do we need to clone a repo here and not just use the repo we are already in?

Reply:
So the general dev workflow would be

1. Create the notebook
2. Push it to a branch
3. Repeat 2. every time you make a change to the notebook

? That is quite inconvenient. Is there no better way to get this file into the cluster whether it is deployed locally or remotely?

In any case, we definitely need to document this workflow in the README.